lingua_franca/lang/parse_common.py

```diff
@@ -385,3 +386,23 @@ def replace_right(source, target, replacement, replacements=None):
             extract = extract_handler(to_parse, short_scale, ordinals)
     numbers.reverse()
     return numbers
+
+
+class DurationResolution(Enum):
+    TIMEDELTA = 0
+    RELATIVEDELTA = 1
+    RELATIVEDELTA_STRICT = 1
```

Comment:
You have two `1`s here. If these are actually equivalent, I would suggest just using one, rather than having two equal values.

Not sure what the target python version is here, but if it's 3.6 or better, I would recommend using [auto](https://docs.python.org/3.8/library/enum.html#enum.auto).